src/Views/gui_view.py

```diff
@@ -558,34 +483,26 @@ def draw_realization(self, start: list[int], end: list[int]) -> None:
         self.draw_triangle(start, end, 'white')
 
     def add_relation(self):
-        # TODO: This will break once our back end is in
-        # Create list of classes in Diagram
         class_options = [cb._name for cb in self._class_boxes]
-
-        dialog = Add_Relation_Dialog(self, class_options, "Add Relationship")
-        if dialog.result:
-            src, dest, rel = dialog.result
-
-            new_command = "add relation " + src + " " + dest + " " + rel
-            self._user_command.set(new_command)
+        rel_options = ["Aggregation", "Composition", "Inheritance", "Realization"]
```

Comment:
we should be able to import these from wherever they currently are. Not necessary, but it would make things change proof. You can import specific non-class objects from other files as well.